error AC1000: Unknown build error: System.NullReferenceException: Object reference not set to an instance of an object

When compiling a project containing a grammar on Visual Studio for Mac, I get this error message:

> error AC1000: Unknown build error: System.NullReferenceException: Object reference not set to an instance of an object

The call stack is:

```
Target Antlr4Compile:
    Executing command: "/private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/../tools/net45/Antlr4.exe" -o obj/Debug/ -encoding UTF-8 -listener -visitor -Dlanguage=CSharp -package My.Sql /private/var/root/ws/mypackage/My.Data/Sql/SQL.g4 /private/var/root/ws/mypackage/My.Data/Sql/SQLTokens.g4
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error: can't Load group file /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/tools/net45/Tool/Templates/Codegen/CSharp/CSharp_v4_5.stg
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error: Caused by: Object reference not set to an instance of an object
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Misc.TemplateRuntimeMessage..ctor (Antlr4.StringTemplate.Misc.ErrorType error, System.Int32 ip, Antlr4.StringTemplate.TemplateFrame frame, System.Exception e, System.Object arg, System.Object arg2) [0x00007] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Misc.TemplateRuntimeMessage..ctor (Antlr4.StringTemplate.Misc.ErrorType error, System.Int32 ip, Antlr4.StringTemplate.TemplateFrame frame, System.Exception e, System.Object arg) [0x00000] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Misc.ErrorManager.RuntimeError (Antlr4.StringTemplate.TemplateFrame frame, Antlr4.StringTemplate.Misc.ErrorType error, System.Exception e, System.Object arg) [0x00013] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.TemplateGroupFile.get_RootDirUri () [0x00021] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.TemplateGroup.ImportTemplates (Antlr.Runtime.IToken fileNameToken) [0x00055] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Compiler.GroupParser.group (Antlr4.StringTemplate.TemplateGroup group, System.String prefix) [0x00132] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.TemplateGroup.LoadGroupFile (System.String prefix, System.String fileName) [0x0008a] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error: can't Load group file /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/tools/net45/Tool/Templates/Codegen/CSharp/CSharp_v4_5.stg
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error: Caused by: Object reference not set to an instance of an object
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Misc.TemplateRuntimeMessage..ctor (Antlr4.StringTemplate.Misc.ErrorType error, System.Int32 ip, Antlr4.StringTemplate.TemplateFrame frame, System.Exception e, System.Object arg, System.Object arg2) [0x00007] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Misc.TemplateRuntimeMessage..ctor (Antlr4.StringTemplate.Misc.ErrorType error, System.Int32 ip, Antlr4.StringTemplate.TemplateFrame frame, System.Exception e, System.Object arg) [0x00000] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Misc.ErrorManager.RuntimeError (Antlr4.StringTemplate.TemplateFrame frame, Antlr4.StringTemplate.Misc.ErrorType error, System.Exception e, System.Object arg) [0x00013] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.TemplateGroupFile.get_RootDirUri () [0x00021] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.TemplateGroup.ImportTemplates (Antlr.Runtime.IToken fileNameToken) [0x00055] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.Compiler.GroupParser.group (Antlr4.StringTemplate.TemplateGroup group, System.String prefix) [0x00132] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.StringTemplate.TemplateGroup.LoadGroupFile (System.String prefix, System.String fileName) [0x0008a] in <3f8c237f3f0f4e919fecc046951e5b22>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error: Unhandled Exception:
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error: System.NullReferenceException: Object reference not set to an instance of an object
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.AbstractTarget.GetRecognizerFileName (System.Boolean header) [0x00022] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.CodeGenerator.GetRecognizerFileName (System.Boolean header) [0x00015] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.OutputModelController.BuildLexerOutputModel (System.Boolean header) [0x0000c] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.CodeGenerator.GenerateLexer (System.Boolean header) [0x00007] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.CodeGenPipeline.Process () [0x00115] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.ProcessNonCombinedGrammar (Antlr4.Tool.Grammar g, System.Boolean gencode) [0x000c2] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.Process (Antlr4.Tool.Grammar g, System.Boolean gencode) [0x000ba] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.ProcessGrammarsOnCommandLine () [0x00062] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.Main (System.String[] args) [0x00014] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error: [ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.AbstractTarget.GetRecognizerFileName (System.Boolean header) [0x00022] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.CodeGenerator.GetRecognizerFileName (System.Boolean header) [0x00015] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.OutputModelController.BuildLexerOutputModel (System.Boolean header) [0x0000c] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.CodeGenerator.GenerateLexer (System.Boolean header) [0x00007] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.Codegen.CodeGenPipeline.Process () [0x00115] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.ProcessNonCombinedGrammar (Antlr4.Tool.Grammar g, System.Boolean gencode) [0x000c2] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.Process (Antlr4.Tool.Grammar g, System.Boolean gencode) [0x000ba] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.ProcessGrammarsOnCommandLine () [0x00062] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
    /private/var/root/ws/mypackage/packages/Antlr4.CodeGenerator.4.6.5-beta001/build/Antlr4.CodeGenerator.targets(121,5): error AC1000: Unknown build error:   at Antlr4.AntlrTool.Main (System.String[] args) [0x00014] in <41572b1990f84ca19e2bc9202ebd76d7>:0 
Done building target "Antlr4Compile" in project "My.Data.csproj" -- FAILED.

```
I have updated ANTLR to the Antlr4.CodeGenerator.4.6.5-beta001 version to get support for generating on the Mac, and I use the new C# implemented code generator using the instructions found here: https://github.com/tunnelvisionlabs/antlr4cs/releases/tag/v4.6.5-beta001.

Is this my fault? How can I fix this?

--- a/runtime/CSharp/Antlr4.Tool/Antlr4.Tool.csproj
+++ b/runtime/CSharp/Antlr4.Tool/Antlr4.Tool.csproj
@@ -43,8 +43,8 @@
 
   <ItemGroup>
     <PackageReference Include="Antlr3.CodeGenerator" Version="3.5.2-beta1" PrivateAssets="all" />
-    <PackageReference Include="Antlr3.Runtime" Version="3.5.1" />
-    <PackageReference Include="StringTemplate4" Version="4.0.8" />
+    <PackageReference Include="Antlr3.Runtime" Version="3.5.2-beta2" />
+    <PackageReference Include="StringTemplate4" Version="4.0.9-beta2" />
   </ItemGroup>
 
   <ItemGroup>

